tests/cbor_decode/test1_suit_old_formats/testcase.yaml

```diff
@@ -0,0 +1,5 @@
+tests:
+  cddl_gen.cbor_decode.test1_suit_old_formats:
+    platform_allow: nrf9160dk_nrf9160 nrf52840dk_nrf52840 nrf52dk_nrf52832 nrf51dk_nrf51422
```

Comment:
Is there any need for these extensive platform_allow lists? We only execute on native posix in CI AFAICS

Reply:
Agreed. This seems unwieldy to maintain across all samples if we can have native posix verify functionality on CI.